eclipse-scout-core/src/tooltip/TooltipSupport.ts

```diff
@@ -134,21 +134,29 @@ export class TooltipSupport {
       return; // removed in the meantime (this method is called using setTimeout)
     }
     let text = this._text($comp);
-    if (!text) {
-      return; // treat undefined and no text as no tooltip
+    if (!text) { // treat undefined and no text as no tooltip
+      this._destroyTooltip();
+      return;
     }
 
+    let $anchor = this._options.$anchor || $comp;
     let htmlEnabled = this._htmlEnabled($comp);
 
     if (this._tooltip && this._tooltip.rendered) {
       // update existing tooltip
+      let anchorChanged = this._tooltip.$anchor !== $anchor;
+      this._tooltip.$anchor = $anchor;
```

Comment:
I would add setters for the properties on the tooltip class and do the position update in set$Anchor to make it work without a tooltip support as well.